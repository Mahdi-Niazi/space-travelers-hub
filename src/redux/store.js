import { configureStore } from '@reduxjs/toolkit';

import missionReducer from './slices/MissionsSlice';

const store = configureStore({
  reducer: {
    missions: missionReducer,
  },
});

export default store;
