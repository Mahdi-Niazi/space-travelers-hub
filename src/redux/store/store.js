import { configureStore } from '@reduxjs/toolkit';
import rocketsReducer from '../slices/RocketsSlice';

const store = configureStore({
  reducer: {
    rockets: rocketsReducer,
  },
});

export default store;
