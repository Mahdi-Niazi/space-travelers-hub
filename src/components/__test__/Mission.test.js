import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MissionList from '../Mission-lists';
import store from '../../redux/store';

describe('Mission component', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <MissionList />
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
