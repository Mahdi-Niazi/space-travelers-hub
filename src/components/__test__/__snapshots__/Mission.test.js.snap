// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission component should match snapshot 1`] = `
<tr>
  <td
    className="titles"
  />
  <td />
  <td
    className="memberBtn"
  >
    <span
      className="nonactiveStyle"
    >
      NOT A MEMBER
    </span>
  </td>
  <td
    className="memberBtn"
  >
    <button
      className="joinMission"
      onClick={[Function]}
      type="button"
    >
      Join Mission
    </button>
  </td>
</tr>
`;
